api/proto/config/devmodel.proto

```diff
@@ -56,3 +86,38 @@ message SystemAdapter {
 	// and used for app instances
 	string logicalName = 6;
 }
+
+enum PhyIoType {
+        PhyIoNoop = 0;
+        PhyIoEth = 1;
```

Comment:
Makes sense to add a comment that Eth really means Networking, or add Wlan and Wwan as separate.

Reply:
Its better to have separate enums. I will add others.

Reply:
Fine with me, but then we should define the set which are networking (because networking can be used for systemAdapters and today we have specific checks for this in Eve by comparing with "CioEth". If such checks aren't needed on the cloud and UI, then we can keep the definition of the set on the Eve side. But I suspect the UI also check which things are ethernets/networks. Makes sense to check that with Sahil. Or we can defer adding the enums until after we check with him